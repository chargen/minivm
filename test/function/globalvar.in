a = 10
b = 20
func foo(x)
  b = 30
  return a * b + x
end

print foo(50)
print a
print b
