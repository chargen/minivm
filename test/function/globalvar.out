350
10
20
